### ISSUE TITLE
Sphinx rendering of parameters section changed?

### ISSUE THREAD

**Issue by sylee957** [sympy/sympy#16585]
https://docs.sympy.org/latest/modules/discrete.html#sympy.discrete.transforms.fft
https://docs.sympy.org/dev/modules/discrete.html#sympy.discrete.transforms.fft

![image](https://user-images.githubusercontent.com/34944973/55685205-52bf2f00-598e-11e9-94a4-afd3b547860a.png)

![image](https://user-images.githubusercontent.com/34944973/55685216-69fe1c80-598e-11e9-973d-7e01d55f9d76.png)

I see the indentation changed, and it looks more disorganized in later one

**Comment by moorepants** [comment]
Sphinx 2.0 recently came out. I noticed quite a few backwards incompatibilities in the changelog, maybe this is related.

**Comment by sylee957** [comment]
As far as I had tried with the maintained numpydoc extension, the problem still remains, and it is a dead end.
I haven't tried with napoleon yet.

**Comment by sylee957** [comment]
Okay, I've also tried with napoleon but things are not working too.

It may possibly be broken from the sphinx side (e.g. issue https://github.com/sphinx-doc/sphinx/issues/6262)
I see the parameters sections are technically rendered as field lists, and it seems like we to change the style to something other than `classic`, or patch our CSS theme to get it working

I've applied the one https://github.com/sphinx-doc/sphinx/pull/6266 manually, and things may not be identical as previous, but seems like working.
![image](https://user-images.githubusercontent.com/34944973/62278567-50279e80-b483-11e9-8b54-0a6c8cf11046.png)
